uber/models.py

```diff
@@ -1324,6 +1348,22 @@ def past_years_json(self):
         return json.loads(self.past_years or '[]')
 
 
+class WatchList(MagModel):
```

Comment:
Picking the first match seems fine, since I can't imagine a deliberate use case for having an attendee who matches two different watchlist entries.  For that reason, I don't see this as a many-to-many relationship so the way it's currently defined seems okay to me.